cpp/tests/CMakeLists.txt

```diff
@@ -210,9 +210,13 @@ if(BUILD_TESTS)
 
   ConfigureTest(NAME NEIGHBORS_BALL_COVER_TEST PATH neighbors/ball_cover.cu GPUS 1 PERCENT 100)
 
+  set(NCCL_DEPS "")
+  if(BUILD_MG_ALGOS)
+    list(APPEND NCCL_DEPS NCCL::NCCL)
+  endif()
   ConfigureTest(
     NAME NEIGHBORS_ALL_NEIGHBORS_TEST PATH neighbors/all_neighbors/test_float.cu GPUS 1 PERCENT 100
-    ADDITIONAL_DEP NCCL::NCCL
+    ADDITIONAL_DEP ${NCCL_DEPS}
```

Comment:
Thanks for finding this! We can just get rid of this NCCL line altogether because this doesn't depend on nccl at all.